src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/PkgImportControlTest.java

```diff
@@ -69,6 +71,13 @@ public void setUp() {
         icRootRegexpParent.addChild(icBootRegexpParen);
     }
 
+    @Test
+    public void testRegexp() {
+        // . replaced with x to avoid matching with regexp meta-character .
+        assertNull(icUncommon.locateFinest("comxkazgroup.courtlink.uncommon.regexp", "MyClass"),
```

Comment:
what do you thing about starting using Truth here?

Reply:
i'll do it in another PR once this is merged